adevinta/vulcan-checks
# Migrate vulcan-exposed-memcached to v2

* Using the port and protocol for the affected resource
* Labelled as issue
* Using the version provided by the memcached server as the finterprint

A small refactor has been done too.

To test tcp:
```
docker run -ti --rm -p11211:11211/tcp --name memcached memcached
```

To test udp:
```
docker run -ti --rm -p11211:11211/udp --name memcached memcached -U 11211
```

local.toml used:
```
[Check]
Target = "localhost"
AssetType = "Hostname"
Options = '{"port":11211}'
[Log]
LogLevel = "DEBUG"
```
